src/sidebar.component.ts

```diff
@@ -76,8 +75,7 @@ import { isBrowser, upperCaseFirst, isLTR, isIOS } from './utils';
       transition: transform 0.3s cubic-bezier(0, 0, 0.3, 1);
     }
   `],
-  changeDetection: ChangeDetectionStrategy.OnPush,
-  encapsulation: ViewEncapsulation.None
```

Comment:
I only removed ViewEncapsulation.None. In my opinion, I prefer to use the default value if it is possible (emulated) as a solution to the future of browsers with shadow sun. Do you think that is better ViewEncapulation.None or Emulated?